src/index.d.ts

```diff
@@ -101,7 +101,7 @@ export type rgbaArray = ReadonlyArray<number>;
 // int32ARGBColor = 0xaarrggbb
 export type int32ARGBColor = number;
 
-export type Color = int32ARGBColor | rgbaArray | string;
+export type Color = int32ARGBColor | rgbaArray | OpaqueColorValue | string;
 
```

Comment:
Could we just replace the "Color" type altogether with "ColorValue" from react native? It should represent the same set of types of colors

Reply:
It's not the same, from the type it's `string | OpaqueColorValue` so we could replace the `string` with this one, but not the whole type. I prefer to use `OpaqueColorValue` here as I'm not sure what types will be added to `ColorValue` in the future.

Reply:
I was under the impression that `ColorValue` is just the standard Color type in React Native 0.63+, and that `OpaqueColorValue` will include all the extra types of colors that can be passed to native.  In react-native-macos (which we recently added support for in react-native-svg), I extended ColorValue/OpaqueColorValue to include a new API `ColorWithSystemEffectMacOS` (https://github.com/microsoft/react-native-macos/pull/751). 

But also, I thought the idea of `OpaqueColorValue` is that we do not need to know the specifics of the type, as long as it gets passed over the bridge to `RCTConvert UIColor:]` (or whatever the equivalent on Android is), React Native will make sure it's a valid native color. Is there a reason we need to know the specifics of the types of colors now? I was under the impression we could abstract most of that away to React Native and would need less processing on this repo?

Reply:
You can find the definition [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-native/index.d.ts#L6077):
```ts
export type ColorValue = string | OpaqueColorValue;
```

So it's not the same, and it can't be replaced as the way this library deals with colors is quite custom. I'm not the author of this library, but i guess it's partly due to color inheritance. If it was not dealing with colors in a custom way then this PR wouldn't be necessary.